FirebaseAppDistribution/Sources/FIRAppDistribution.m

```diff
@@ -71,8 +71,8 @@ - (instancetype)initWithApp:(FIRApp *)app appInfo:(NSDictionary *)appInfo {
 
   if (self) {
     [GULAppDelegateSwizzler proxyOriginalDelegate];
-    self.UIService = [FIRAppDistributionUIService sharedInstance];
-    [GULAppDelegateSwizzler registerAppDelegateInterceptor:self.UIService];
```

Comment:
@jeremydurham UIService or uiService? What's the convention? I decided to change it back to UIService earlier based on Jeremy's feedback.

Reply:
I think Cleo (and you) are right. uiService makes sense (similar to httpResponse)